src/theme/foundations/typography.ts

```diff
@@ -174,7 +174,6 @@ const typography: Typography = {
         subtitle2: fontSizeValues.mobile["subtitle2"],
       },
     },
-    // deprecated semantic tokens
     breadcrumbs: {
       // The deprecated value is "caption" but we should use body2 instead
       // for backwards compatibility
```

Comment:
I'm not exactly sure what this note means, but do we need it?

Reply:
I'm not sure either. I wrongly removed the breadcrumbs styles because of the comment but it turns out that removing it was wrong.

@bigfishdesign13 should we keep the comment or add more for clarity?

Reply:
This can be removed.

For understanding, the original `caption` token was 14px font size. And that was the token that was used in the `Breadcrumbs` component. When we updated the typography, the new `caption` token was 12px, while `body2` was 14px. To keep the font size the same in the component, `body2` was necessary. 

Yes, that comment should have provided more details. But I will say, it made COMPLETE sense when I wrote it.